<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContact</class>
 <widget class="QDialog" name="AddContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Partner ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>-999999999</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Public Address:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="recvr">
       <property name="plainText">
        <string>-----BEGIN PUBLIC KEY-----
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzEWkZs7v2vg5lk7BnbMl
7/Y5zh1N4lsfdoCfs3CXzI578mzvWCM6Wwi8Gm5zgTmff2W4cxsZg5a03GztcjJW
e6U5INegUl8N0Q/ck+ulQIFqi47Q/gOPvI38qF/evPHBhSIEoVSYv9V3hsZT6p5K
PGBkhaWNzYLrXgSS5AhTLDTMk7w7SPSOLNgcsUR3K3wd2DUMjvC+olCz/IMMCuTS
Xnc7RsR2efQ0p3XDVior4saMjw9nqL8LPqT6IcHYmZCNhdWWz0YS+IMPIzbJApVm
g/kPsiaa0ys508gNHLLGuiCKIOatAzXdm+ZI+FIgUrO7cSKKNyT6zSi1CufGTz4f
KQIDAQAB
-----END PUBLIC KEY-----</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
